### PR TITLE
Feature/visual studio installation

### PR DESCRIPTION
**Pull request by thinking-tower**
See #108 

I've added a guide for Visual Studio 2019 with `example_with_all.cpp`. I copied all files (`example_with_all.cpp`, `crow_all.h`, `vcpkg.json`) to the same root folder to not complicate things. 

I've tested this with `example_vs.cpp` and `example_chat.cpp` too (`crow.h` -> `crow_all.h`, copy `example_chat.html` if needed).

### CODE REVIEW

File: vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        {
+            "name": "boost-lexical-cast",
+            "version": "1.70.0" 
+        },
+        {
+            "name": "boost-optional",
+            "version": "1.70.0" 
+        }
+    ],
+    "builtin-baseline": "44d94c2edbd44f0c01d66c2ad95eb6982a9a61bc"
```
Comment (The-EDev) [inline vcpkg.json:70]: I was wondering what `builtin-baseline` did, just for me to be able to maintain this file later on.

**Review by The-EDev — APPROVED** [review]
As someone who never developed using C++ on windows, I was able to follow the instructions quite easily, with the only bump being setting up an empty C++ project (because I forgot to install the Desktop C++ development package). I got a working build, both debug and release. Although one problem was that Visual studio gave me red lines over references to crow (things like `json is not defined in crow`). Other than that everything seems great.

Thank you very much for your help! If no one else has any comment by the end of today I'll merge your PR.

**Review by ayaankhan98 — APPROVED** [review]
LGTM! Thanks @thinking-tower

File: vcpkg.json (hunk 1 above)
Comment (thinking-tower) [reply vcpkg.json:70]: That's the commit hash of the `vcpkg` repo.  See [here](https://vcpkg.readthedocs.io/en/latest/users/versioning/#builtin-baseline).

Comment (The-EDev) [reply vcpkg.json:70]: understood, thanks

**Pull request merged**